test/rubygems/test_gem_spec_fetcher.rb

```diff
@@ -168,20 +168,20 @@ def src.fetch_spec(name)
   def test_suggest_gems_from_name_latest
     spec_fetcher do|fetcher|
       fetcher.spec "example", 1
-      fetcher.spec "other-example", 1
```

Comment:
This gem should stay as a non-matching example with too much distance.

We should also add a test that matches only this gem, like `gem install other-apple`